kolibri/plugins/learn/assets/src/composables/useLearnerResources.js

```diff
@@ -321,7 +321,7 @@ export default function useLearnerResources() {
    * @public
    */
   function fetchResumableContentNodes() {
-    const params = { resume: true, max_results: 12 };
+    const params = { resume: true, max_results: 12, ordering: '-last_interacted' };
```

Comment:
This is correct, although we seem to be missing the parallel change that was made to the `HomeHydrate` viewset in `kolibri/plugins/learn/viewsets.py`?

Reply:
Are you able to throw more lights on this?

Reply:
When we were looking at this together, you made updates in this file, at this line: https://github.com/learningequality/kolibri/blob/release-v0.15.x/kolibri/plugins/learn/viewsets.py#L229 and this line: https://github.com/learningequality/kolibri/blob/release-v0.15.x/kolibri/plugins/learn/viewsets.py#L233 to add the `ordering` to the `LearnHomePageHydrationView` as well, but this seems to be missing from the pull request.